chevah/chevah-keycert
# Create a SSL CSR

Problem
======

We want to create CSR + key from command line but also from Python API

Changes
=======

Created 3 new methods. one to generate the commnad line options, another one to to generate the key/csr and another one to take care of writing to disk.


As a drive by I have updated the test runner to print speed reports and wrote a test for self signed certificate so that we have 100% coverage for ssl module.

How to test
========

reviewers: @alibotean 

check that changes make sense

Use the demo tool to generate a csr
